draft-thomson-http-oblivious.md

```diff
@@ -302,7 +302,7 @@ identifier for an AEAD.
 expressed using the TLS syntax; see {{Section 3 of TLS}}.
 
 ~~~ tls-syntax
-opaque HpkePublicKey<1..2^16-1>;
+opaque HpkePublicKey[Npk];
```

Comment:
Oh, I just realized that this isn't right as we need the KEM ID to precede this.